config/mainnet.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/spacemeshos/go-spacemesh/bootstrap"
 	"github.com/spacemeshos/go-spacemesh/checkpoint"
 	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/config/util"
```

Comment:
I couldn't place it in the `config` package because there would be cyclic dependencies (i.e. `config -> activation` and `activation -> config`). I can rename it, but how? `github.com/spacemeshos/go-spacemesh/config/types`?

Reply:
`github.com/spacemeshos/go-spacemesh/config/types` works for me 👍